src/opamLexer.mll

```diff
@@ -111,6 +111,7 @@ rule token = parse
 | "("    { LPAR }
 | ")"    { RPAR }
 | '\"'   { STRING (buffer_rule string lexbuf) }
+| "\"\"\"\n" { STRING (buffer_rule string_triple lexbuf) }
```

Comment:
Is this correct? I thought that the point with #18 was that we should neither print nor ignore a newline at the start?

Reply:
ping @AltGr

Reply:
Hm, maybe a stale fix that I had laying around concurrently to #18 ? I don't remember :thinking: